DESCRIPTION

```diff
@@ -1 +1,13 @@
-Config/renv/profiles/reporting/dependencies: quarto, leaflet, plotly, sf
+Config/renv/profiles/dev/dependencies:
+    DBI,
+    igraph,
+    lubridate,
+    openxlsx,
+    readr,
+    rmarkdown,
+    RJDBC
```

Comment:
These are the _additional_ dependencies needed for `00-ingest.R`, `07-export.R`, `08-api.R`, and building `README.Rmd`.